examples/support/analysis/config/service-monitor.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/component: prometheus
+    app.kubernetes.io/instance: k8s
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 2.39.1
+  name: prometheus-k8s
+  namespace: podtato-kubectl
```

Comment:
is this correct? 🤔 In the markdown we only used `analysis-demo` as ns
the same for the rest of resources

Reply:
nvm, you refer to a different example for this one :)